db/schema.rb

```diff
@@ -535,6 +535,7 @@
     t.datetime "updated_at", null: false
     t.datetime "flagged_value_updated_at"
     t.index ["ip_address", "representative_id", "flag_type", "flagged_value_updated_at"], name: "index_unique_constraint_fields", unique: true
+    t.index ["ip_address", "representative_id", "flag_type"], name: "index_unique_flagged_veteran_representative", unique: true
```

Comment:
Should there be a migrate file for this index addition?

Reply:
that is interesting... not my change, i'll rebase the schema!

Reply:
fixed